framework/db/CDbConnection.php

```diff
@@ -567,16 +567,17 @@ public function getLastInsertID($sequenceName='')
 	/**
 	 * Quotes a string value for use in a query.
 	 * @param string $str string to be quoted
+	 * @param integer $quoteParam Parameter for PDO::quote function. Default : PDO::PARAM_STR
 	 * @return string the properly quoted string
 	 * @see http://www.php.net/manual/en/function.PDO-quote.php
 	 */
-	public function quoteValue($str)
+	public function quoteValue($str, $quoteParam = PDO::PARAM_STR)
```

Comment:
I used the same method signature as PDO::quote(string, integer). I don't understand what does not look right ?